src/wallet/mod.rs

```diff
@@ -1556,7 +1556,7 @@ where
 
         let sync_time = SyncTime {
             block_time: BlockTime {
-                height: maybe_await!(self.client.get_height())?,
+                height: maybe_await!(self.client.get_height())? as u32,
```

Comment:
Maybe this field `BlockTime pub height: u32` can be changed to `u64` in other PR ?

Reply:
It can be, then we have to change height to u64 in all of wallet code. That means everywhere. That felt more intrusive to me at this stage, so basically just contained the conversion only in blockchain level. This is where it it gets converted back to u32 for rest of the lib.

I have scoured few other projects (rust-bitcoin, rust-lightning) and there is no consistent type for height. Both of these use u32, so as bdk.